addons/hr_expense/models/hr_expense.py

```diff
@@ -292,6 +292,10 @@ def _compute_from_product_id_company_id(self):
             expense.name = expense.name or expense.product_id.display_name
             expense.product_uom_id = expense.product_id.uom_id
             expense.tax_ids = expense.product_id.supplier_taxes_id.filtered(lambda tax: tax.company_id == expense.company_id)  # taxes only from the same company
```

Comment:
:weary: 
Stored compute fields need to have their own computed method, otherwise
```python
class MyModel(models.Model):
    _name = 'my.model'
    fieldA = fields.Char(compute='_compute_fields', store=True, readonly=False)
    fieldB = fields.Char(compute='_compute_fields', store=True, readonly=False)

    def _compute_fields(self):
        for record in self:
            record.fieldA = 'a'
            record.fieldB = 'b'
```
```python
env['my.model'].create({'fieldA': 'c'})
# `fieldA` will be 'a' instead of 'c' because 
# `_compute_fields` needs to be called to compute `fieldB`
```